modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-publish.adoc

```diff
@@ -0,0 +1,103 @@
+= Anypoint MQ Publish Operation - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Publish operation enables you to create a new Anypoint MQ message and send it to the specified destination: queue, FIFO queue, or message exchange. With it, you can configure both the content of the message and all the headers and properties you need.
+
+== Outgoing Message Structure
+
+Messages contain these parameters:
+
+* `body`
++
+Contains the body of the published Anypoint MQ message.
+* `properties`
++
+Represents a simple key/value map of Strings that are the message properties of the message.
```

Comment:
GLOBAL "key/value" > "key-value"    I'll stop marking from here onward in this file.

Why is Strings init. capped?